test/test_framework.py

```diff
@@ -1255,8 +1255,8 @@ class SomeObject(Object):
             _stored = StoredState()
 
         class WrappedFramework(Framework):
-            def __init__(self, store, charm_dir, meta, model):
-                super().__init__(store, charm_dir, meta, model)
+            def __init__(self, store, charm_dir, meta, model, event_name):
```

Comment:
Should we make this optional as in `event_name=None` to match `Framework`'s arg, and avoid the "foo"s below?